### DISCUSSION TITLE
How the model storage aborescence works behind the model registry APIs ?

### DISCUSSION THREAD

**Discussion by nthimotte8** [mlflow/mlflow#6159] [Q&A] [answered]
Hi,

I want to understand how the model storage aborescence works behind the model registry APIs.

I wanted to know if the stageworks like a tag,
if the . pickel model is moved to different branches depending on the stage.
same question for versions ?

sincerely,

nicolas

**Comment by JHibbard** [comment, marked as answer]
Hi nthimotte8,

I may be misunderstanding your question, but I think you're asking where are model artifacts stored? The answer varies depending on your deployment scenario and configuration. I'd refer you to the MLflow docs [[here](https://mlflow.org/docs/latest/tracking.html#how-runs-and-artifacts-are-recorded)]. Depending on the scenario you may save model artifacts to a local file directory structure or to a remote blob storage.

Hope that helps!
Jim
